### DISCUSSION TITLE
Using the real-world Star systems

### DISCUSSION THREAD

**Discussion by ghost** [endless-sky/endless-sky#10447] [Ideas]
I want to see 'endless-sky' pilot their own assembled spaceship, exploring and developing other planets.

- List of nearest galaxies https://en.wikipedia.org/wiki/List_of_nearest_galaxies real-world Star systems.
---

### off-topic
https://exoplanetarchive.ipac.caltech.edu/ NASA confirmed planets
https://en.wikipedia.org/wiki/Observable_universe

youtube
[Size Comparison of the Universe](https://www.youtube.com/watch?v=134nAVodmp8)
[ABSOLUTE Size Comparison 2023 (FULL) 4K 60FPS 3D](https://www.youtube.com/watch?v=WocfYpjgb8M&t=5576s)
 space ship https://youtu.be/watch?v=8OrZX347MoE

**Comment by mOctave** [comment]
If you wanted, you could make an overhaul plugin (or possibly a fork) doing this. I know some people have tried things along these lines (more specifically, changing the map so that systems are in real-world positions), but never to the scale of multiple galaxies. It would certainly be an ambitious project, but I would play it.

That said, I don't expect this would ever be vanilla.

**Reply by bene-dictator** [reply]
It's impossible to represent the 3D universe in Endless Sky's 2D map. I have tried different ways, but never with good results.

I have been trying to add some real-life planets (or references) to them in ES, e.g. Independance, Dagon Station.

**Reply by Zireael07** [reply]
What ways did you try? I am developing a 2D space game with real-world maps and I find that using x,y directly and representing Z with color works well

**Reply by bene-dictator** [reply]
Well, one way was just not representing the Z axis at all. Trying to get the correct positions involved a lot of changing between galactic co-ordinate systems, it was rather annoying. Colour wouldn't work because they're used for gov'ts in ES. Maybe a better way was to incorporate the Z axis with a 2.5D representation (like you see on the 30ly diagramme of star systems nearby Sol, or the ones from Project Rho/Atomic Rockets) but figuring out hyperlinks got tough because it's already difficult to tell which star is closest to another in a mess of overlapping lines and stars.

**Reply by Zireael07** [reply]
Project Rho was a BIG inspiration for my own maps! 

Re: gov'ts, I understand gov'ts is just one of many map modes in ES, why couldn't Z be another?

**Reply by bene-dictator** [reply]
Too much effort to do in ES :james:

> Project Rho was a BIG inspiration for my own maps!

It's an encyclopaedia, I swear. Brilliant website. ES goes a lot of its well-argued points, though, ~~like ship insurance being crucial~~. I'm definitely going to be using it for some things I'm working on.

**Reply by TheGiraffe3** [reply]
> Re: gov'ts

Because there are already so many map modes, maybe?..

**Comment by TheGiraffe3** [comment]
Developing other planets would be very difficult to do with the current mission syntax. There is a Terraforming Hope plugin mission chain.
Maybe made by 1010todd, but I'm not really sure.

The way we have sprites will make it almost impossible for FL to build their own.
Also see #10277 that somewhat touched on ship-building.
